events/hfe_events_primitive.txt

```diff
@@ -37,6 +37,7 @@ country_event = {
 				set_owner = root
 				create_ship = {
 					random_existing_design = colonizer
+					colonizer_species = owner_species
```

Comment:
I do not use owner_species and such scopes, it is very old
Betetr to use for example root.species
Also is this string really nessesary?

Reply:
Yes. It created a Colony Ship without a species. What is the difference between "ROOT.SPECIES" and "OWNER_SPECIES" ?

"OWNER_MAIN_SPECIES" was removed from the game.

Reply:
Things like owner_species is very old artifact.
Dot switches is newer thing and better to use it.
As newer thing it should work better (like faster or less bags, who know) and have a lower chance to be removed with some patch.
Also mod's style and consistency.

Reply:
"OWNER_SPECIES" works perfectly and does not cause lag. Honestly, it's better than "OWNER.SPECIES", although that has its purposes sometimes. No change is needed unless Paradox breaks it.

Paradox did, however, remove "OWNER_**MAIN**_SPECIES".

Reply:
My preference is "OWNER_SPECIES". To my knowledge, Paradox uses it too.